server/server_test.go

```diff
@@ -105,32 +105,13 @@ func (seq *callSeq) GetNumber(ctx context.Context, call air.CallSequence_getNumb
 	return nil
 }
 
-type lockCallSeq struct {
```

Comment:
Is this just an unrelated drive-by "oh look, this is dead code," or is this related the the primary change in this pr in some way I'm missing?

Reply:
Correct, it's just dead code that was flagged by the linter.  I figured it wouldn't add too much noise to the PR (and then forgot to mention it).